### PR TITLE
This is my code for the header-nav

### LINKED ISSUE #2: Header Section
the **header section** contains the following: 

* Logo 
* Header nav which contains:
    - Home
    - Services
    - Projects
    - Team 
    - About

### PR DESCRIPTION
**Pull request by Ola014**
Relates #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <title>Athathy.com</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+          <h1 class="company-name">Athathy</h1>
+          <nav class="header-nav">
+            <ul class="header-nav-list">
+              <li><a href="./index.html">Home</a></li>
```
Comment (sara219) [inline index.html:17]: You already have a folder for images, why you put it in the same location as the file.

Move all images to the specified folder

File: index.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }
+    .header-nav{
+        width: 100%;
+    }
+    .company-name{
+        padding-bottom: 10px;
+        margin-left: 0px;}
+    .header-nav-list a {
+        font-size: 15px;
+        }
```
Comment (sara219) [inline index.css]: missing closing **`}`** for the media query

File: index.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    justify-content: space-around;
+    align-items: center;
+}
+.header-nav-list a{
+    text-decoration: none;
+    color: white;
+    font-size: 22px;
+}
+.header-nav-list a:hover{
+    color:#71585b;
```
Comment (sara219) [inline index.css]: the shadow colour make the hover uncomfortable to look at, try to remove it or chose sth else 
Screenshot from 2021-03-01 20-01-42

File: index.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    justify-content: space-between;
+    align-items: center;
+    background-color:#b89252  ;
+    opacity: 0.9;
+    padding: 15px ;
+}
+.company-name{
+    color: white;
+    margin-left: 20px;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (sara219) [inline index.css:30]: this font family didn't change anything from the default one, import it from [google font  ](https://fonts.google.com/)

File: index.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    margin: 0;
+    box-sizing: border-box;
+}
+.header{
+    color:white;
+    background-image: url(header.jpg);
+    background-repeat: no-repeat;
+    background-position:center ;
+    background-size:cover ;
+    height: 600px;
```
Comment (sara219) [inline index.css]: Why did you add the showcase here even though this is a header section?!!

Delete it from here,
you should add everything in its particular section.

File: index.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    color:#71585b;
+    font-weight: bold;
+    text-shadow: 2px 2px 5px #011800 ;
+}
+.show-case{
+    background-image: url(header.jpg);
+    background-repeat: no-repeat;
+    background-position:center ;
+    background-size:cover ;
+}
```
Comment (sara219) [inline index.css]: repeated code ( the header above), and also not related to the header section #2

**Review by sara219 — CHANGES_REQUESTED** [review]

File: index.css (hunk 4 above)
Comment (Ola014) [reply index.css:30]: Ok . I will remove the font family. Arial is good font that is convenient for website design,

File: index.css (hunk 6 above)
Comment (Ola014) [reply index.css]: That's a mistake from me, I will remove it.

File: index.css (hunk 5 above)
Comment (Ola014) [reply index.css]: This is a background image for both header-nav and showcase section. I can not delete it.

File: index.css (hunk 4 above)
Comment (Ola014) [reply index.css:30]: This is a convenient font family for this website design.

**Review by sara219 — APPROVED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by ahmedhamouda2 — APPROVED** [review]

**Pull request merged**